config/config.exs

```diff
@@ -59,6 +59,10 @@ if System.get_env("NEW_RELIC_LICENSE_KEY") != "" do
 
   config :elixometer, reporter: :"Elixir.Exometer.NewrelicReporter",
     update_frequency: 60_000
+
+  config :new_relic_agent, app_name: "Spacesuit #{Mix.env}",
```

Comment:
You know this already reports to New Relic using the NewRelicReporter, right? See the line above.

Reply:
Yep sure, but the intent was to use `new_relic_agent` module instead because it's official, actively developed and will allow us to instrument more in the future.